client/src/modules/payroll/rubrics/modals/rubric.modal.js

```diff
@@ -0,0 +1,77 @@
+angular.module('bhima.controllers')
+  .controller('RubricModalController', RubricModalController);
+
+RubricModalController.$inject = [
+  '$state', 'RubricService', 'ModalService', 'NotifyService', 'appcache',
+];
+
+function RubricModalController($state, Rubrics, ModalService, Notify, AppCache) {
+  var vm = this;
+  vm.rubric = {};
+
+  var cache = AppCache('RubricModal');
+
+  if ($state.params.creating || $state.params.id) {
+    vm.stateParams = cache.stateParams = $state.params;
+  } else {
+    vm.stateParams = cache.stateParams;
+  }
+  vm.isCreating = vm.stateParams.creating;
+
+  vm.selectAccountFour = function selectAccountFour(account) {
+    vm.rubric.third_party_account_id = account.id;
+  };
+
+  vm.selectAccountSix = function selectAccountSix(account) {
+    vm.rubric.costs_account_id = account.id;
+  };
+
+  vm.setDiscount = function setDiscount(value) {
+    vm.setting = true;
+    vm.rubric.is_discount = value === 1 ? true : false;
```

Comment:
Can you use `ng-true-value` and `ng-false-value` to do this automatically in the view?